app/components/Checkbox.tsx

```diff
@@ -0,0 +1,44 @@
+import type { PropsWithChildren } from "react";
+
+export function Checkbox(
+    props: {
+        name: string;
+        value: string;
+        checked?: boolean;
+        "aria-describedby"?: string;
+    } & PropsWithChildren,
```

Comment:
Is there a reason why you are using PropsWithChildren and not passing props to it? Semi confused. I also don't even see where children is being rendered...